wall_e/src/resources/cogs/reminders.py

```diff
@@ -49,22 +52,38 @@ def __init__(self, bot, config):
             logger.error("[Reminders __init__] enountered following exception when setting up PostgreSQL "
                          "connection\n{}".format(e))
 
-    @commands.command()
+    @commands.command(aliases=['remindmeon'])
     async def remindmein(self, ctx, *args):
         logger.info("[Reminders remindmein()] remindme command detected from user {}".format(ctx.message.author))
         parsed_time = ''
         message = ''
+        tz = _timezone
         parse_time = True
         for index, value in enumerate(args):
             if parse_time:
                 if value == 'to':
                     parse_time = False
                 else:
+                    if value in pytz.all_timezones:
+                        tz = pytz.timezone(str(value))
                     parsed_time += "{} ".format(value)
```

Comment:
if I am not mistaken, the code goes over this line once so I say, below line 63, print what `parsed_time` and `tz` ends up being set to.

Reply:
I think it already logs what parsed_time is (see line 112), but I'll log the timezone too